Libplanet/Store/IStore.cs

```diff
@@ -14,7 +14,7 @@ public interface IStore
         /// <returns>Existing namespaces.</returns>
         IEnumerable<string> ListNamespaces();
 
-        long CountIndex(string @namespace);
+        int CountIndex(string @namespace);
```

Comment:
If this change is merely for being along with `CountTransactions()`/`CountBlocks()`, we'd better make these three methods to all return `long` instead.